intersect_registry_service/app/main.py

```diff
@@ -75,6 +76,14 @@ async def lifespan(app: FastAPI) -> typing.AsyncGenerator[None, None]:
 add_logging_middleware(app)
 app.add_middleware(CorrelationIdMiddleware)
 
+app.add_middleware(
+    SessionMiddleware,
+    secret_key=settings.SESSION_SECRET,
+    max_age=60 * 60 * 24 * 7,
```

Comment:
So am I reading this right as a 1 week max age?
Do we need to make that configurable or does it matter?

Reply:
We can make it configurable. Session can be pretty long for a lot of sites, although longer sessions typically employ the session ID approach detailed above. (By long, I mean like a couple of years).